apps/web/components/Embed.tsx

```diff
@@ -175,14 +180,16 @@ const Codes: Record<string, Record<string, (...args: any[]) => string>> = {
       return code`
 import Cal, { getCalApi } from "@calcom/embed-react";
 import { useEffect } from "react";
-function MyComponent() {
+export default function MyApp() {
```

Comment:
Changed it to this, as it makes it easier to just copy paste and get it running on CodeSandbox. I guess it would be easier for others too.